web_src/js/markup/math.ts

```diff
@@ -3,7 +3,7 @@ import {displayError} from './common.ts';
 function targetElement(el: Element) {
   // The target element is either the current element if it has the
   // `is-loading` class or the pre that contains it
-  return el.classList.contains('is-loading') ? el : el.closest('pre');
+  return el.classList.contains('is-loading') ? el : (el.closest('pre') ?? el.closest('code'));
```

Comment:
Comment above needs update too.

Reply:
Then it needs a deep cleanup. 87f87c7a2c7aeb3f05afb449f523804dc4444c83